python/interpret_text/experimental/unified_information/unified_information_explainer.py

```diff
@@ -125,16 +128,16 @@ def explain_local(self, text, classes=None, predicted_label=None, true_label=Non
         self._optimize(num_iteration, lr=0.01, show_progress=True)
         local_importance_values = self._get_sigma()
         self.local_importance_values = local_importance_values
+        # predicted_label = self._wrapped_model.predict([X])
```

Comment:
commented out code

Reply:
I will remove. I just don't know the safest way to create the predictions.